mount/mount.c

```diff
@@ -35,11 +66,15 @@ int main() {
 	const char *mnt_ns = getenv("MOUNT_NS");
 	const char *net_ns = getenv("NET_NS");
 	const char *source = getenv("MOUNT_SOURCE");
+	const char *nfs_mount_hostname = getenv("MOUNT_NFS_HOSTNAME");
 	const char *target = getenv("MOUNT_TARGET");
 	const char *flags = getenv("MOUNT_FLAGS");
 	const char *options = getenv("MOUNT_OPTIONS");
 	const char *fs_type = get_fs_type();
 
+	char final_options [200];
+	strcpy(final_options, options);
```

Comment:
Maybe use strncpy and add an assertion, or make this buffer something impossibly large, like 4096 (PAGE_SIZE)?